client/stylesheets/puzzle.scss

```diff
@@ -297,66 +297,74 @@ table.puzzle-list {
 // Puzzle metadata
 .puzzle-metadata {
   flex: none;
-  padding-bottom: 2px;
+  padding: 2px 4px;
 }
 
 .puzzle-metadata-row {
   display: flex;
-  min-height: 26px;
-  padding: 0 4px;
-  vertical-align: middle;
+  width: 100%;
+  font-size: 14px;
+  line-height: 28px;
+  align-items: flex-start;
+  align-content: flex-start;
+  justify-content: space-between;
 }
 
-.puzzle-metadata-tag-editor-row {
-  min-height: 46px;
-  align-items: center;
+.puzzle-metadata-title-set {
+  flex:1;
 }
 
-.puzzle-metadata-left {
-  flex: 1 1 auto;
-  padding: 4px;
-  order: 1;
+.puzzle-metadata-title {
+  font-weight: bold;
 }
 
-.puzzle-metadata-right {
-  flex: 1 .2 auto;
-  margin: 4px;
-  text-align: right;
-  order: 2;
+.puzzle-metadata .view-count {
+  cursor: pointer;
 }
 
-.puzzle-metadata-tags {
-  display: inline-block;
-  height: 24px;
+.puzzle-metadata-answer, .puzzle-metadata .tag {
```

Comment:
Any reason you're overriding the styles for tags here?  I definitely agree with the adjustments to make the answer field larger and rounded, but applying the same to `.puzzle-metadata .tag` makes the tags look inconsistent with tags everywhere else in the app.

I'd suggest:

1. make this block not apply to `.puzzle-metadata .tag`
2. decide if you want to change the tag styles everywhere, and if you do, update `.tag` accordingly (~line 142) if so so we only have one set of styles for that type of widget

Reply:
I changed it in this context to make line heights and alignment consistent in the metadata pane.  I was reluctant to change tags globally, for fear of breaking something somewhere else.  It looks like tags only show up on the landing page and the related puzzles pane, so it probably wouldn't be too bad to make sure everything is consistent.

Reply:
Cool, happy to have rounded tags everywhere.  I think I originally made them the way they are to minimize pixels given to padding and to maximize information density, but I don't think we're quite that strapped for space.  I'd say try making this new style be the one we use everywhere then?

Reply:
Sounds good to me.

Reply:
Good call on this.  Turned out to be pretty easy to do and the consistency is undeniably nice.